edison/static/js/policy.model.js

```diff
@@ -28,6 +25,7 @@ export class Policy {
         }
 
         this.condition = this.condition + add_condition;
+        this.#countCondition++
     }
```

Comment:
maybe the two rows above this comment should be inside the if statement?
because it will be wrong to add something to the counter or to the condition itself if the add_condition is an empty string

Reply:
the if statement is if I added a first condition or more, if I add the second or more I first added 
 a comma to the condition string and then adding the new condition

Reply:
hooo i missed the fact that you asked about the condition itself and not about the add_condition.
closing this conversation :)